testing/expected-m.go

```diff
@@ -173,3 +173,26 @@ func CheckDate(expectedStr string, format string) func(i interface{}) error {
 		return errors.New(fmt.Sprintf("Expected date %s but got %s", expected, found))
 	}
 }
+
+func CheckDateClose(target time.Time, leeway time.Duration) func(interface{}) error {
+	return func(val interface{}) error {
+		if val == nil {
+			return errors.New(fmt.Sprintf("Expected date %s +/- %s", target, leeway))
```

Comment:
error string syntax does not follow guidelines see: https://github.com/golang/go/wiki/CodeReviewComments#error-strings